data/json/items/tools.json

```diff
@@ -8259,7 +8259,13 @@
     "ammo": "NULL",
     "revert_to": "null",
     "category": "veh_parts",
-    "use_action": "HORN_BICYCLE"
+    "use_action": {
+        "type": "manualnoise",
+        "use_message": "You honk the %s.",
```

Comment:
The "%s" in this string is never replaced.

The string is used at:
```C++
p->add_msg_if_player( _(use_message.c_str()) );
```

But there is no additional parameter that can be used to replace the "%s", can either replace the "%s" in JSON (where you already know what item is invoked), or add a parameter to `add_msg_if_player`.